cmd/manager/main.go

```diff
@@ -81,9 +81,10 @@ func main() {
 
 	printVersion()
 
-	if !environment.IsOperator() {
-		// Take exclusive instance lock, we need that to make sure that
-		// we don't have more than one working instance of k8s-nmstate
+	// Lock only for handler, we can run old and new version of
+	// webhook without problems, policy status will be updated
+	// by multiple instances.
+	if environment.IsHandler() {
```

Comment:
The fact that we need this kind of `if` several times in the main function suggests we should maybe create a function mainWebhook, mainHandler, mainOperator. They would share some logic, but the result would be way more readable. If somebody wants to follow execution of a webhook, they just jump to that function.

Reply:
Will do a follow up PR.